setup/Directory.Build.targets

```diff
@@ -10,4 +11,11 @@
   -->
   <Target Name="IncludeTransitiveProjectReferences" >
   </Target>
+
+  <!-- Copies the vsix files after they've been signed to the different output folders. -->
+  <Target Name="CopyVsixAfterSigning" DependsOnTargets="AddVsixForSigning" AfterTargets="SignFiles">
+    <Copy SourceFiles="$(OutDir)$(AssemblyName).vsix" DestinationFolder="$(VisualStudioSetupInsertionPath)" />
+    <Copy SourceFiles="$(OutDir)$(AssemblyName).vsix" DestinationFolder="$(VisualStudioSetupOutputPath)" />
+  </Target>
```

Comment:
Do these copies happen when building locally in VS? Ideally they wouldn't do, right?

Reply:
This target is for the setup projects (the ones that build the NuGet packages). Is it normal to run those locally? I think that might happen when running `build.cmd`. I can create a binlog and take a look.

Reply:
Looking into it, the nupkg files do get built when you run `build.cmd`. I looked at the binlog and these new targets do not get ran. However, the reason that they don't run is interesting. The `SignFiles` target only exists when the MicroBuild signing plugin is installed. Therefore, the binlog shows this:
> The target "SignFiles" listed in a BeforeTargets attribute at "C:\Code\project-system\src\Directory.Build.targets (18,42)" does not exist in the project, and will be ignored.

So, the reason these targets don't run because `SignFiles` itself doesn't exist locally. So, that's some good knowledge to move forward with. I think keeping this implementation now is fine. However, one of the goals of cleaning up the build process would be reducing the amount of file copying (which is extensive at the moment). I'll put this information into an issue when I make one for replacing the RepoToolset Vsix process. This copying only exists because it needs to feed into that.

Reply:
Created an issue for this here: https://github.com/dotnet/project-system/issues/7869